tooling/lib/test/index.js

```diff
@@ -7,8 +7,8 @@ const debug = require('debug')('tooling:test');
 
 const dotenv = require('dotenv');
 
-dotenv.config({path: '.env.default'});
```

Comment:
wait, really? how has anything ever worked? :)

Reply:
I'm guessing manual env vars having the actual keys you need